### ISSUE TITLE
TWE: IS22: RP020: HfLA Website Team Members Exit Interview Recording Upload Form: Create

### ISSUE THREAD

**Issue by pandanista** [hackforla/internship#440]
### Dependencies
- [x] #365  has been created

### Overview
We need to create a new form to upload the recordings made for RP020.

During these tasks you will be asked to add link to the Resources/Instructions section, and that section is at the bottom of this issue (go view it now).

### Action Items
- [x] UX lead asks admin to add the team member working on this issue to the 1Password internship-UX's vault
- [x] UX lead adds the assignee to the Internship - PII's "My Drive" as a Contributor so that they have access to internship - PII's My Drive where the recordings and transcriptions with PII are stored.
   - [x] UX lead accesses the Internship - PII's "My Drive"
   - [x] Choose "Manage members", which is located towards the top right side of the browser <img width="200" alt="Screenshot 2023-08-09 at 6 09 36 PM" src="https://github.com/hackforla/internship/assets/57029070/f7cc7fbe-1b00-4ded-8b44-b10fdbccae46">
   - [x] Enter the assignee's email address and select the role as "Contributor" <img width="300" alt="Screenshot 2023-08-09 at 6 11 40 PM" src="https://github.com/hackforla/internship/assets/57029070/41ab3531-75b1-435a-836a-1b24be3ecb69">
   - [x] Confirm with the assignee that they have access to the Internship - PII's "My Drive" 
  
#### Customize Resource Links
- Customize Resource for this Research Plan's Wiki Page
    - [x] Go to Wiki Research Output Overview page (Resources # 1.01)
    - [x] Go to the `Research by Plan Number` section
    - [x] Choose the `Research Plans, sorted by number` link
    - [x] Choose `RP020` 
    - [x] After it directs you to the RP020's wiki page, copy the URL from the browser window
    - [x] Update Resources # 2.01 by pasting the URL your just copied into parentheses at the end of the line with no space between `]` and `(` so it becomes a hyperlink
- Customize Resource for this Research Plan's Google Drive Folder
    - [x] Open the Google Drive's Research by Participant Type Folder (Resources  # 1.02)
    - [x] Open the `HfLA Volunteers` Folder
    - [x] Copy the link for the `RP020` folder
        1. Right-click on the `RP020` folder
        1. Choose "Share"
        1. Choose "Copy link"
    - [x] Update Resource # 2.02 with the link you just copied. Paste into parentheses at the end of the line so it becomes a hyperlink.
    - [x] Choose "Update comment" in Github and make sure all the checkboxes above have been checked
- [x] Open the `RP020` folder (Resources # 2.02) if not already open
    - [x] Copy the link to the `TWE: IS22: RP020: HfLA Website Team Members Exit Interview Roll Call and Session Table` file
        1. Right-click on the `TWE: IS22: RP020: HfLA Website Team Members Exit Interview Roll Call and Session Table` folder
        1. Choose "Share"
        1. Choose "Copy link"
    - [x] Update Resources # 2.03 by pasting the link you just copied inside parentheses at the end of the line so it becomes a hyperlink
    - [x] Choose "Update comment" in Github and make sure all the checkboxes above have been checked  

####  Set up a folder for the recording upload form
- Customize Resource for Interview Recording Folder stored on the Internship - PII's `My Drive`
    - [x] Log into your Google account so you will be able to access the Internship - PII's 'My Drive' in the next steps
    - [x] Open the `TWE Interview Recordings by Plan #` folder by choosing the URL linked in Resources # 1.03
    - [x] Create a new folder for RP020's recording uploads in the `TWE Interview Recordings by Plan #` folder by choosing "New" and "New folder" on the left side bar <img width="127" alt="Screenshot 2023-01-27 at 2 34 43 PM" src="https://user-images.githubusercontent.com/57029070/215219196-2e8b1712-6c74-4893-a1aa-7db51ca48e19.png"> <img width="200" alt="Screenshot 2023-01-27 at 2 35 00 PM" src="https://user-images.githubusercontent.com/57029070/215219378-052eefb3-64e5-447f-941d-1c2d04f3f1f2.png">       
    - [x] Name the "New folder" by copying and pasting: 
    ```
    TWE: IS22: RP020: HfLA Website Team Members Exit Interview Recordings
    ```
    - [x] Copy the link to the `TWE: IS22: RP020: HfLA Website Team Members Exit Interview Recordings` folder 
        1. Right-click on the `TWE: IS22: RP020: HfLA Website Team Members Exit Interview Recordings` folder
        1. Choose "Share"
        1. Choose "Copy link"
    - [x] Update Resources # 2.04 by pasting the copied link inside parentheses at the end of the line so it becomes a hyperlink
    - [x] Choose "Update comment" in Github and make sure all the checkboxes above have been checked 

####  Set up a temporary folder for receiving the RP[Replace 000]'s recording uploads in internship-ux@hackforla.org's `My Drive`
- [x] Log in the internship-ux@hackforla.org's Google Drive in a separate Chrome/browser profile window (you do not want to cross-contaminate this process with your personal Google account profile) so you can access the Google Form template in the next steps
   - If you are not sure how to do this, use the following steps to log in:
      1. Log into 1password (Resources # 1.04) 
           - Refer to the guide linked in Resources # 1.04.01 if you are unsure about any of the following steps
      1. Choose internship-UI/UX vault
      1. Choose <img width="150" alt="Screenshot 2023-10-05 at 1 12 13 PM" src="https://github.com/hackforla/internship/assets/57029070/290a2046-e430-4b4f-a99f-62b4721bb142"> to see login info for internship-ux@hackforla.org account
      1. In your Chrome web browser, select your Google Account profile picture in the top right corner <img width="95" alt="Screenshot 2023-06-19 at 5 45 54 PM" src="https://github.com/hackforla/internship/assets/57029070/959e62bd-c6f7-413a-a87f-b94cb05a4bcd">
      1. Under "Other profiles", choose "+ Add" 
      1. A "Set up your new Chrome profile" window will pop up. Choose "Sign in"
      1. Copy and paste the internship-ux@hackforla.org login info from 1password vault to set up your new profile. Resources # 1.04.01 has step by step instructions to guide you through this.
      1. Once logged in, choose "Drive" among the Google apps listed under the profile                                                                     <img width="300" alt="Screenshot 2023-06-19 at 5 57 46 PM" src="https://github.com/hackforla/internship/assets/57029070/658942eb-39dc-482d-92d7-10d683fe13ea">
- [x] Create a new folder for RP020's recording uploads in internship-ux@hackforla.org's `My Drive` by choosing "New" and "New folder" on the left side bar <img width="127" alt="Screenshot 2023-01-27 at 2 34 43 PM" src="https://user-images.githubusercontent.com/57029070/215219196-2e8b1712-6c74-4893-a1aa-7db51ca48e19.png"> <img width="200" alt="Screenshot 2023-01-27 at 2 35 00 PM" src="https://user-images.githubusercontent.com/57029070/215219378-052eefb3-64e5-447f-941d-1c2d04f3f1f2.png">       
    - [x] Name the "New folder" by copying and pasting: 
    ```
    TWE: IS22: RP020: HfLA Volunteers Exit Interview Recording Uploads Temporary Folder
    ```
    - [x] Copy the link to the `TWE: IS22: RP020: HfLA Volunteers Exit Interview Recording Uploads Temporary Folder`
        1. Right-click on the `TWE: IS22: RP020: HfLA Volunteers Exit Interview Recording Uploads Temporary Folder`
        1. Choose "Share"
        1. Choose "Copy link"
    - [x] Update Resources # 2.05 by pasting the copied link inside parentheses at the end of the line so it becomes a hyperlink
    - [x] Choose "Update comment" in Github and make sure all the checkboxes above have been checked

####  Make a Copy of the Google Form Template for RP020 in Internship-ux@hackforla.org's `My Drive`
- [x] Make sure you are logged in as the internship-ux@hackforla.org Google profile
- [x] Open the Google Form Template link (linked in Resources # 1.05) in the internship-ux@hackforla.org browser window
    1. Copy link from Resources # 1.05
    1. Open a new browser tab in the internship-ux@hackforla.org browser window
    1. Paste the link into the address bar
- [x] Choose the `Use Template` button
- [x] A message that says `Missing File Upload folders` pops up again, choose `Restore` create a destination folder for the file uploads ![Restore](https://user-images.githubusercontent.com/57029070/211466563-f7bf3944-5db5-4e99-8729-05938af56244.png) 
- A `Folders restored successfully` notification should flash at the bottom of the window
- [x] Move the form you just created into the `TWE: IS22: RP020: HfLA Volunteers Exit Interview Recording Uploads Temporary Folder` by completing the following steps
    1. Choose the `Move to folder` icon ![Screenshot 2023-01-09 at 6 34 19 PM](https://user-images.githubusercontent.com/57029070/211449094-c5479c29-0351-486c-837f-d9cca4321aba.png)
    1. Click on the back arrow
    1. Choose `My Drive`
    1. Choose `TWE: IS22: RP020: HfLA Volunteers Exit Interview Recording Uploads Temporary Folder`
    1. Choose `Move Here`
    1. Choose `Move`
    1. Choose `Dismiss` on the alert notification
- Rename the Google Form you just created by following the steps:
    - [x] Open the Google Form if it is not open already
    - [x] Copy
    ```
    TWE: IS22: RP020: HfLA Website Team Members Exit Interview Recording Upload Form
    ``` 
    - [x] Paste the text you copied into the top field of the form you just created to replace the `TWE: Template: IS22: RP020: HfLA Website Team Members Exit Interview Recording Upload Form`
    <img width="600" alt="Screenshot 2023-01-27 at 2 52 03 PM" src="https://user-images.githubusercontent.com/57029070/215221211-c2f7fc32-ddaa-4ed8-b094-33623162af09.png">
#### Customize the Recording Upload Google Form just made
- In the Google Form's body text of "Finishing up: Sending your recording to TWE team" section, complete the following edits:
    - [x] Copy
    ```
    RP020-U000-HfLAWTM000
    ``` 
    - [x] Paste the copied text in the section after "Rename the recording file in the following format" to replace `**_RP[Replace 000]-UX Researcher Abbreviation###-[Replace TYPE OF PARTICIPANT] Abbreviation###_**` (see the grayed out texts in the screenshot below)  <img width="600" alt="Screenshot 2023-01-17 at 4 46 59 PM" src="https://user-images.githubusercontent.com/57029070/213053453-d8a93e94-b0f8-444c-95ea-0c4e423d88de.png">
    - [x] Make sure the text you just copied are in bold and italic in the Google Form
    - [x] Choose `[Replace TEMPLATE: Interview Roll Call and Session Table]` in the google form, and choose on the pencil icon that popped up to edit link <img width="435" alt="Screenshot 2023-06-19 at 6 44 11 PM" src="https://github.com/hackforla/internship/assets/57029070/dd1ea305-2f6e-4bd9-8c79-f81f75ebce6e">
    - [x] Copy the URL from Resource # 2.03
    - [x] In the "Link to" section, paste the link you just copied
    - [x] Copy
    ```
    TWE: IS22: RP020: HfLA Website Team Members Exit Interview Roll Call and Session Table
    ```
    - [x] In the "Text to display" section, paste what you just copied into the line <img src="https://user-images.githubusercontent.com/57029070/211993361-ad9cd48f-52ba-4c1d-9e22-07e05a6c8e32.png" width="500">
    - [x] Choose "OK"
    - [x] Copy 
    ```
    3. HfLAWTM### (An example: Intern No. 2 is I002)
    ```
    - [x] Paste what you copied into question 3 to replace the original texts 
- In the form edit, go to Settings
    - [x] Under Responses section, choose "Verified" as the drop down answer to the `Collect email addresses` setting so we know who the sender is
    - [x] Make sure to toggle off the `Restrict to users in Hack for LA and its trusted organizations` option <img src="https://user-images.githubusercontent.com/57029070/211475661-ebad4f04-d6fa-48e8-a325-59346a163ee0.png" width="700" height="50">
    - [x] Choose `Send` button (upper right of the window)
    - [x] Choose link <img src="https://user-images.githubusercontent.com/57029070/211476642-a4d010c8-a1ef-475f-809c-3e73095272ed.png" width="60" height="30">
    - [x] Check the `Shorten URL` box, then choose `Copy`
    - [x] Update Resource # 2.06 with the link you just copied. Place it into parentheses at the end of the line so it becomes a hyperlink. 
    - [x] Choose "Update comment" in Github and make sure all the checkboxes above have been checked

#### Edit Response Destination
- In the form edit, choose `Responses`
- [x] Choose 3 vertical dots   <img width="30" alt="Screenshot 2023-01-31 at 5 48 33 PM" src="https://user-images.githubusercontent.com/57029070/215925141-2a3f8145-08cc-4ba9-bbe2-507aed50e8b0.png">
- [x] Choose `Select destination for responses`
- [x] Choose `Create a new spreadsheet`
- [x] Choose `Create`
- [x] Open the `TWE: IS22: RP020: HfLA Volunteers Exit Interview Recording Uploads Temporary Folder` (Resources # 2.05)(You may already have this open)
- [x] Update the spreadsheet link in Resources # 2.07
    - [x] Copy link of the `TWE: IS22: RP020: HfLA Website Team Members Exit Interview Recording Upload Form (Responses)` spreadsheet
        1.  Right-click on the `TWE: IS22: RP020: HfLA Website Team Members Exit Interview Recording Upload Form (Responses)` spreadsheet
        1. Choose "Share"
        1. Choose "Copy link" 
    - [x] Update Resources # 2.06 by pasting the link inside parentheses at the end of the line so it becomes a hyperlink
- Rename the folder you just restored in the internship-ux@hackforla.org's `My Drive`
    - [x] Go to `TWE: IS22: RP020: HfLA Volunteers Exit Interview Recording Uploads Temporary Folder`
    - [x] Locate the folder you restored with the name `Finishing up: Sending your recording to TWE team (File responses)` in the drive
    - [x] Copy
    ```
    TWE: IS22: RP020: HfLA Website Team Members Exit Interview Recording Upload Form (File responses)
    ```
    - [x] Right-click on the `Finishing up: Sending your recording to TWE team (File responses)` folder, and choose `Rename`
    - [x] Paste the texts you copied into the text box, and apply `OK`
    - [x] Update the receiving folder link in Resources # 2.08
        - [x] Choose 3 vertical dots on the right side of the folder <img width="73" alt="Screenshot 2023-09-12 at 6 07 08 PM" src="https://github.com/hackforla/internship/assets/57029070/d356c842-9048-4c09-99cf-85ecbeae94a0">
         1. Choose "Share"
         1. Choose "Copy link" 
    - [x] Update Resources # 2.08 by pasting the link inside parentheses at the end of the line so it becomes a hyperlink
- [x] Choose "Update comment" in Github and make sure all the checkboxes above have been checked off
- [x] Sign out of the internship-ux@hackforla.org Google account

#### Update Research Plan Wiki Page 
- [x] Open `Wiki: Research Plan 020` (Resources # 2.01)
- [x] Choose "Edit" to edit the wiki page
- [x] Find the "Interview Recordings & Transcripts" section
- [x] Add a new line and insert a "-" dash
- [x] Copy and paste Resource # 2.06 (both the text and URL) at the end of this new line
- [x] Choose "Save Page"
- [x] Open the Google drive folder RP020 (Resources # 2.02) 
- [x] Check that `TWE: IS22: RP020: HfLA Website Team Members Exit Interview Recording Upload Form` IS NOT saved in this folder
- [x] Review with UX Lead
- [ ] Product sign-off
- [ ] UXR Lead or PM removes the assignee from the Internship - PII's My Drive if the issue has been reviewed and the deliverables are approved
- [ ] When closing this issue, UX lead or PM needs to create a new issue to track the receival of all the video recordings in the temporary folder (Resources # 2.05) in internship-ux@hackforla.org's `My Drive` once the interviews are conducted.
- [ ] After all the recordings are uploaded into the temporary folder, UX Lead or PM needs to move the recordings to `TWE: IS[Replace YY]: RP[Replace 000]: [Replace TYPE OF PARTICIPANT] [Replace TYPE OF INTERVIEW] Interview Recordings` folder (linked in Resources # 2.04) in internship - PII's `My Drive` for data privacy compliance.

#### Resources for creating this issue
1.01 [Wiki: Research Output Overview](https://github.com/hackforla/internship/wiki/Research-Output-Overview)
1.02 [Google Drive Folder: Research by Type](https://drive.google.com/drive/folders/1f5Qgq-ikT_UwcgRBuoBamqY0Wacsg9f5?usp=share_link)
1.03 [TWE Interview Recordings by Plan #](https://drive.google.com/drive/folders/1lvP_k4qGKD05QzbHBtIpEl4B3X7ylzoh?usp=sharing) _This folder is only accessible from Internship - PII's "My Drive"_
1.04 [1password](https://start.1password.com/signin?l=en)
    1.04.01 [Guide: Log into a Google Account with 2-Step Verification Using 1Password](https://docs.google.com/document/d/1hL3gYqoV6EWl4bUa2VYLgq0N__CjTvSw4Da2M4x2Hxk/edit?usp=sharing) _If this is your first time logging into Internship's private drive, please follow the instructions to log in._
1.05 [Interview Recording Upload Form Template](https://docs.google.com/forms/d/1P-pVycbrwwnf1KM0QH1MPkxvkBAUYNnADbchik0IbdI/template/preview) _This template is only accessible using internship-ux@hackforla.org's Google account_

#### Resources gathered during the completion of this issue
2.01 [Wiki: Research Plan 020](https://github.com/hackforla/internship/wiki/Research-Plan-20:-Intern-Interaction-with-HfLA-Website-Team)
2.02 [Google Drive Folder: RP020](https://drive.google.com/drive/folders/1Mt099BzsWqhBe-y02ReDIEsRHB7vNdWR?usp=drive_link)
2.03 [TWE: IS22: RP020: HfLA Website Team Members Exit Interview Roll Call and Session Table](https://docs.google.com/spreadsheets/d/1Gl4RK35YOeSwjuShfAQsKlgGSiEM7Es_MZ7kD72m_sQ/edit?usp=drive_link)
2.04 [TWE: IS22: RP020: HfLA Website Team Members Exit Interview Recordings](https://drive.google.com/drive/folders/19UDUwZMjK2J9V52-QqaDNPFZDACR0vrd?usp=drive_link) _accessible from Internship - PII's My Drive_
2.05 [TWE: IS22: RP020: HfLA Volunteers Exit Interview Recording Uploads Temporary Folder](https://drive.google.com/drive/folders/13s485o5MGYRdxMDwcczwFnTSiYF4qHhL?usp=drive_link) _accessible from Internship-UX's My Drive_
2.06 [TWE: IS22: RP020: HfLA Website Team Members Exit Interview Recording Upload Form](https://forms.gle/2RzNKSfubwCuExja9)  _accessible from Internship-UX's My Drive_
2.07 [TWE: IS22: RP020: HfLA Website Team Members Exit Interview Recording Upload Form (Responses)](https://docs.google.com/spreadsheets/d/17cVyHGfmWfcrFSkM_0ZpnPsI15llizS6Xcjh-U4BtEQ/edit?usp=drive_link)  _accessible from Internship-UX's My Drive_
2.08 [TWE: IS22: RP020: HfLA Website Team Members Exit Interview Recording Upload Form (File responses)](https://drive.google.com/drive/folders/13eUD9j7ySg8iY8uFZ5R84q-mx49RgsgWFjW5_gTPhBmyew1YZO19CV_hgSQNBI_uZm8tAcMQ?usp=drive_link)  _accessible from Internship-UX's My Drive_

**Comment by pandanista** [comment]
@jessieliu1219 Thank you for taking up this issue. I am asking the admin to add you to the 1password vault. 

FYI: This is a testing template. Since the issue was created based on a template we are still testing—meaning, while working on this issue, you might run into instructions that are not clear enough, or confusing, or direct you to a wrong folder, etc. Therefore, please leave any feedback you have below as comments as you check off the action items. Much appreciated. :) 

Please see below for instructions on providing feedback and examples:
### Action Items for Providing Feedback on Testing Templates
- [ ] Provide feedback for this issue
     - [ ] Add feedback in the form of (1) questions and/or (2) explicit instructions for how the content should be improved. Feedback for GitHub templates, please screenshot the section that needs to be modified and provide written feedback in a comment in this issue thread. Feedback in a Google Doc or spreadsheet should be given in the form of document comments. 
     - [ ] Determine the accuracy of the content for each section (action items listed in the issue above, Google doc, or Google sheet). If you find an inaccuracy, error, or mis-categorization, explain how the information presented could be improved.
     - [ ] For each suggested improvement, provide a link, reference, or a screenshot within your comment to justify your suggestion.
        - Examples:  
        <img width="700" alt="Screenshot 2023-03-06 at 6 37 40 PM" src="https://user-images.githubusercontent.com/57029070/223306823-2a79b9df-c996-4ac7-b1b0-e8aeaf0e9170.png">
        <img width="700" alt="Screenshot 2023-03-06 at 6 37 10 PM" src="https://user-images.githubusercontent.com/57029070/223306826-0d29a792-c4fa-4458-96ab-3614b0b66a76.png">
        <img width="700" alt="Screenshot 2023-03-06 at 6 36 56 PM" src="https://user-images.githubusercontent.com/57029070/223306827-ee6c9cad-9fd4-476f-9e8f-bde64a2da151.png">
        <img width="700" alt="Screenshot 2023-03-06 at 6 36 46 PM" src="https://user-images.githubusercontent.com/57029070/223306831-4105863d-192d-4860-810f-efb17cc15e44.png">
- [ ] Once all the steps have been completed, reach out to the current UX Lead to determine if they can incorporate the feedback in the issue template.

**Comment by jessieliu1219** [comment]
There's no link for 2.01 WIKI RESEARCH PLAN 20

**Comment by jessieliu1219** [comment]
I noticed that the file is no longer available for 2.05 after I copied following the instruction (SEND-short URL). I am wondering is this what we intended to do? I assume to make the file available we just need to copy the URL from the webpage but not from the SEND.
<img width="784" alt="Screen Shot 2023-08-15 at 11 13 05 AM" src="https://github.com/hackforla/internship/assets/116132470/c3f8d776-d7e4-4657-a253-f7a0e181d3ab">

**Comment by pandanista** [comment]
@jessieliu1219 Thank you for all the work. I haven't figured out the Google form issue yet, but I will bring it to the meetings and see if the PMs can help troubleshoot it. 

While waiting on the final review, would you please pick up a new issue from the prioritized backlog column? Thanks.

**Comment by pandanista** [comment]
The google form needs to be moved to internship-ux@hackforla.org instead of internship@hackforla.org since it is for internal UX usage.

**Comment by sunannie27** [comment]
@jessieliu1219 
Please provide update
1. Progress: "What is the current status of your project? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures or links* (if necessary): "Add any pictures or links that will help illustrate what you are working on."

* remember to add links to the top of the issue if they are going to be needed again.

**Comment by pandanista** [comment]
@sunannie27 This issue has been reviewed and the deliverables (see the links in the section named "Resources gathered during the completion of this issue") are ready to be reviewed by product.

This is the second testing for template #341. I've incorporated all the changes needed and #341 is ready to be tested by product.

**Comment by sunannie27** [comment]
@pandanista all the links look good to me!
